homeassistant/components/southern_company/sensor.py

```diff
@@ -0,0 +1,305 @@
+"""Support for Southern Company sensors."""
+from __future__ import annotations
+
+import datetime
+from datetime import timedelta
+import logging
+
+import southern_company_api
+
+from homeassistant.components.recorder import get_instance
+from homeassistant.components.recorder.models import StatisticData, StatisticMetaData
+from homeassistant.components.recorder.statistics import (
+    async_add_external_statistics,
+    get_last_statistics,
+    statistics_during_period,
+)
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfEnergy
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .config_flow import InvalidAuth
+from .const import DOMAIN as SOUTHERN_COMPANY_DOMAIN
```

Comment:
No need to alias it.

```suggestion
from .const import DOMAIN
```